site/content/extend/plugins/webapp/best-practices.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Best Practices
+date: 2019-06-02T00:00:00-00:00
+subsection: Web App Plugins
+weight: 0
+---
+
+This page provides best practices for web app plugins.
+
+## User Experience
+
+### Plugin placement in the user interface
+
+1. Actions that apply to specific channels
+- Recommendation: Have your plugin register the actions to [the channel header](https://developers.mattermost.com/extend/plugins/webapp/reference/#registerChannelHeaderButtonAction). This makes it quickly accessible by users and the actions apply to the channel they're viewing. You can additionally [register a slash command](https://developers.mattermost.com/extend/plugins/server/reference/#API.RegisterCommand) on the server-side to take channel-specific actions.
```

Comment:
"to the channel header" sounds awkward to me in english. Perhaps "on the channel header"?

But as an aside, I wonder if this reads like advice on how to make a plugin work for a specific channel, e.g. a button to show in the JIRA channel only or some such. Perhaps a title of, "Actions that apply to a channel"